app/lib/constants/visibility.rb

```diff
@@ -0,0 +1,4 @@
+module Constants::VisibilityTypes
```

Comment:
This can just be Visibility

```suggestion
module Constants::Visibility
```